package.json

```diff
@@ -6,7 +6,8 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "test": "jest"
+    "test": "jest",
```

Comment:
I think I'd prefer to see:

* jest - run jest
* lint - run lint (don't default to fix)
* prettier - run prettier (no args)
* test - `npm run prettier -- --check && npm run lint && npm run jest`